tests/PhakeTest.php

```diff
@@ -1652,6 +1652,36 @@ public function testVerifyNoOtherInteractionsWorks()
         Phake::verifyNoOtherInteractions($mock);
     }
 
+    public function testVariadicVerifyNoOtherInteractions()
+    {
+        $mocks = [Phake::mock('PhakeTest_MockedInterface'), Phake::mock('PhakeTest_MockedClass')];
+        $mocks[0]->foo('a');
+        $mocks[0]->foo('b');
```

Comment:
With the current test, if the current `break` after the first `foreach`, or if you run this test with the old version, the test will pass. To make this test more robust, I would remove this line to make sure that the element that sends the exception is the second one.

Reply:
I added a provider to test multiple test cases in order to cover all cases (I think all).
Please tell me if you need any other test case.